tasks/check/warnings.rake

```diff
@@ -0,0 +1,10 @@
+namespace :check do
+  Rake::TestTask.new(:warnings) do |t|
+    t.libs << "lib"
+    t.ruby_opts << "-r./spec/spec_helper"
+    t.ruby_opts << "-W2"
```

Comment:
What does the `-W2` option signify?
How is it different from just a `-w`?

Reply:
I'm not sure there is fundamentally a difference, at least in this case. Running `ruby --help` produces in part the following output:

```
-w              turn warnings on for your script  -W[level=2]     set warning level; 0=silence, 1=medium, 2=verbose
```

The output looks the same to me. I'd argue that maybe `-W2` is a tiny bit more expressive but I'd be lying if I was saying that with very much confidence.

Reply:
Ah! So `-w` is *probably* equivalent to `-W1`.
Since the level for `-W` defaults to `2`, `-W` is equivalent to `-W2`, correct?

Either ways, I think a comment in the rake task would help future readers:
```ruby
# Output verbose warnings
t.ruby_opts << "-W"
```